pkg/controller/jenkins/configuration/base/resources/pod.go

```diff
@@ -65,7 +65,7 @@ func GetJenkinsMasterContainerBaseEnvs(jenkins *v1alpha2.Jenkins) []corev1.EnvVa
 	envVars := []corev1.EnvVar{
 		{
 			Name:  "JENKINS_HOME",
-			Value: jenkinsHomePath,
+			Value: getJenkinsHomePath(jenkins),
```

Comment:
Please delete those lines and add JENKINS_HOME in NewJenkinsMasterContainer function.